src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -847,22 +847,39 @@ def enable_xformers_memory_efficient_attention(self):
 
         Warning: When Memory Efficient Attention and Sliced attention are both enabled, the Memory Efficient Attention
         is used.
+
+        Parameters:
+            attention_op (`Callable`, *optional*):
+                Override the default `None` operator for use as `op` argument to the
+                [`memory_efficient_attention()`](https://facebookresearch.github.io/xformers/components/ops.html#xformers.ops.memory_efficient_attention)
+                function of xFormers.
+
+        Examples:
+        ```py
+        from diffusers import DiffusionPipeline
+        from xformers import ...  # some attention op
+
+        pipe = DiffusionPipeline.from_pretrained("CompVis/stable-diffusion-v1-4")
+        pipe.enable_xformers_memory_efficient_attention(attention_op=...)
+        ```
```

Comment:
```suggestion
        Examples:

        ```py
        >>> from diffusers import DiffusionPipeline
        >>> from xformers.ops import MemoryEfficientAttentionFlashAttentionOp

        >>> pipe = DiffusionPipeline.from_pretrained("CompVis/stable-diffusion-v1-4").to("cuda")
        >>> pipe.enable_xformers_memory_efficient_attention(attention_op=MemoryEfficientAttentionFlashAttentionOp)
        ```
```